qa/rpc-tests/mining_shielded_coinbase.py

```diff
@@ -34,7 +34,7 @@ def start_node_with(self, index, extra_args=[]):
             nuparams(HEARTWOOD_BRANCH_ID, 10),
             nuparams(CANOPY_BRANCH_ID, 20),
             nuparams(NU5_BRANCH_ID, 20),
-            "-nurejectoldversions=false",
+            "-nurejectoldversions=false"
```

Comment:
No need to remove this trailing comma:
```suggestion
            "-nurejectoldversions=false",
```